### PR TITLE
fix some Scrutinizer issues

### LINKED ISSUE #668: fix Scrutinizer issues
https://scrutinizer-ci.com/g/owncloud/mail/issues/master

@DeepDiver1975 @wurstchristoph @zinks- @Xenopathic let’s make it clean! :)

### PR DESCRIPTION
**Pull request by wurstchristoph**
partially fixes #668

### CODE REVIEW

File: lib/service/html.php [hunk 1]
```diff
@@ -72,6 +73,12 @@ public function filter(&$uri, $config, $context) {
 
 class Html {
 
+	/**
+	 *
```
Comment (LukasReschke) [inline lib/service/html.php]: :question:

**Comment by LukasReschke** [comment]
Besides nitpick :+1: :smile: :rocket:

File: lib/service/html.php (hunk 1 above)
Comment (wurstchristoph) [inline lib/service/html.php]: whoops... removed ;-)

**Comment by jancborchardt** [comment]
Please check out these code improvements @clementhk @zinks- @DeepDiver1975 :)

**Comment by zinks-** [comment]
:+1:

**Comment by wurstchristoph** [comment]
@LukasReschke @zinks- I resolved merge conflicts, please review again ;-)

**Comment by clementhk** [comment]
nice fixes

**Pull request merged**